cmd/fluxctl/history_cmd.go

```diff
@@ -40,7 +40,7 @@ func (opts *serviceHistoryOpts) RunE(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	events, err := opts.API.History(noInstanceID, service)
+	events, err := opts.API.History(noInstanceID, service, time.Now().UTC(), -1)
```

Comment:
This is the wall-clock time at the client, which may be quite different from the time at the service. Could we use a sentinel value to mean "just the latest" instead?